### ISSUE TITLE
[RPM] Not detect libuuid on opensuse

### ISSUE THREAD

**Issue by DevDorrejo** [jgraph/drawio-desktop#493]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Describe the bug**
Hello, i was trying to install drawio-desktop on opensuse tw/leap, and give me that not found libuuid, but is installed:
![image](https://user-images.githubusercontent.com/7697566/107176106-d68c3d00-69a4-11eb-9864-089b7ccc6132.png)

**To Reproduce**
1. Download the RPM installer
2. Try to install
3. Warning dependency about libuuid

**Expected behavior**
smoothly install

**Screenshots**
![image](https://user-images.githubusercontent.com/7697566/107176161-fd4a7380-69a4-11eb-9578-a31e78e0ddbc.png)

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version [e.g. 12.6.7]

**Desktop (please complete the following information):**
 - OS: OpenSUSe TW (last-snapshot) / Leap 15.2

**Additional context**
if i continue breaking the dependency and execute the program will work normally.

**Comment by davidjgraph** [comment]
We do not have the resources to set this environment up and investigate. If nobody is going to look, we'll close the issue.

**Comment by dacog** [comment]
Hi @davidjgraph
I just found this issue, and it is still happening in openSUSE Tumbleweed.  I did some research, and it seems to be the same problem other electron-based project have and adding the following to package.json  (in this case electron-builder-linux-mac.json) seems to make it work. 

```json 
"rpm":{
      "depends":[
        "/usr/lib64/libuuid.so.1"
      ]
    },
 ```
 
 As seen here: https://github.com/BoostIO/BoostNote-App/pull/1228/files
 
 I would be willing to test the change (and make a PR) and it helps and  if it makes sense for you. As I saw on the README "Open-source, not open-contribution" I am asking first :smile:
